packages/cactus-plugin-ledger-connector-ethereum/src/test/typescript/integration/geth-contract-deploy-and-invoke-using-keychain-v1.test.ts

```diff
@@ -240,14 +240,18 @@ describe("Ethereum contract deploy and invoke using keychain tests", () => {
           secret: "",
           type: Web3SigningCredentialType.GethKeychainPassword,
         },
-      });
-      fail("Expected deployContract call to fail but it succeeded.");
-    } catch (error) {
-      console.log("deployContract failed as expected");
-    }
+      })
+    ).rejects.toThrow();
+    
+    console.log("deployContract failed as expected");
+    
   });
 
   test("deployContract with additional parameters should fail", async () => {
+
+    //did not refactor because the test is not actually failing 
```

Comment:
@ashnashahgrover OK, but then let's assert the success with Jest instead and remove the try-catch.

Reply:
See:

https://github.com/hyperledger/cacti/issues/3487